index.html

```diff
@@ -60,6 +60,29 @@
     {% if rgbBlue < rgbMin %}
         {% assign rgbMin = rgbBlue %}
     {% endif %}
+
+    {% if rgbRed <= 0.03928 %}
+        {% assign lRed = rgbRed | divided_by: 12.92 | times: 0.2126 %}
+    {% else %}
+      {% assign lRed = rgbRed | times: 0.3053 | plus: 0.6822 | times: rgbRed | plus: 0.0125 | times: rgbRed | times: 0.2126 %}
+    {% endif %}
+    {% if rgbGreen <= 0.03928 %}
+        {% assign lGreen = rgbGreen | divided_by: 12.92 | times: 0.7152 %}
+    {% else %}
+        {% assign lGreen = rgbGreen | times: 0.3053 | plus: 0.6822 | times: rgbGreen | plus: 0.0125 | times: rgbGreen | times: 0.7152 %}
+    {% endif %}
+    {% if rgbBlue <= 0.03928 %}
+        {% assign lBlue = rgbBlue | divided_by: 12.92 | times: 0.0722 %}
+    {% else %}
+        {% assign lBlue = rgbBlue | times: 0.3053 | plus: 0.6822 | times: rgbBlue | plus: 0.0125 | times: rgbBlue | times: 0.0722 %}
+    {% endif %}
+    {% assign luminance = lRed | plus: lGreen | plus: lBlue %}
+    {% if luminance >= 0.83 %}
```

Comment:
`0.5` seems like a logical cutoff, but I'm not sure if its the best. I made a screenshot so you can see it:
![screenshot](https://user-images.githubusercontent.com/3742559/28920008-414d7158-7850-11e7-9769-a3c7b90b0d3a.png)

I think we can identify two problems from this screenshot:
- First, imo there are a few icons that don't need to be darkened. This includes most yellow icons except for Pingdom, Snapchat and Javascript, and Upwork, Kickstarter and maybe even Gumtree from the green icons.
- Second, it seems a bit odd to me that there are some white icons in between dark icons? Not sure if the icons are not properly ordered or if it is due to the formula that I'm using 🤔 

One last thing, if you want to play around with this more effectively I recommend you check out the [develop-see-luminance](https://github.com/ericcornelissen/simple-icons/tree/develop-see-luminance) branch I created in my fork where instead of the hex-value the calculated luminance is shown so you get a better idea of what icons will be made dark if you change the value (also the cutoff is set to `0.5` on that branch).